tests_integration/conftest.py

```diff
@@ -37,6 +37,12 @@ def invoke(self, *a, **kw):
         return super().invoke(self.app, *a, **kw)
 
     def invoke_with_config(self, *args, **kwargs):
+        return self.invoke(
+            ["--config-file", self.test_snowcli_config, *args[0]],
+            **kwargs,
+        )
+
+    def invoke_with_config_and_integration_connection(self, *args, **kwargs):
```

Comment:
I think we will only use this method, so maybe rename this one to `invoke` and remove `invoke_with_config`? WDTY?

Reply:
We can't use only this method because there are commands which do not have `-c`/`--connection` option.

Reply:
* `snowpark function init`
* `snowpark function package`

Reply:
we should add this option if it's necessary for those commands

Reply:
These commands work offline, they don't need any connection.

Reply:
We can define default connection now, this invoke will not be needed soon. Leave as it is